fastlane/swift/Fastlane.swift

```diff
@@ -2065,7 +2065,7 @@ func jira(url: String,
                                                                                                                 RubyCommand.Argument(name: "initial_build_number", value: initialBuildNumber),
                                                                                                                 RubyCommand.Argument(name: "team_id", value: teamId),
                                                                                                                 RubyCommand.Argument(name: "team_name", value: teamName)])
-  return parseInt(fromString: runner.executeCommand(command))
+  return parseInt(fromString: runner.executeCommand(command).replacingOccurrences(of: "\"", with: ""))
```

Comment:
This file is automatically generated, you'll probably need to take a look at the ruby code in `swift_fastlane_function.rb` or `swift_fastlane_function.rb`